documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -2,54 +2,508 @@
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal,
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal,
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal
+      hard.goals: >
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal,
+        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal  
+      # ...
+----
+
+IMPORTANT: Ensure that the supported `goals`, `default.goals`, and (unless `skipHardGoalCheck` is set to `true`) proposal-specific `spec.goals` include all hard goals specified in `hard.goals` to avoid errors when generating optimization proposals. 
+Hard goals must be included as a subset in the supported, default, and proposal-specific goals.
+
+.Example `KafkaRebalance` configuration for proposal-specific goals
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaRebalanceApiVersion}
+kind: KafkaRebalance
+metadata:
+  name: my-rebalance
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  goals:
+    - RackAwareGoal
+    - TopicReplicaDistributionGoal
+  skipHardGoalCheck: true
+----
+
+=== Skipping hard goal checks
+
+If `skipHardGoalCheck: true` is specified in the `KafkaRebalance` custom resource, Cruise Control does not verify that the proposal-specific goals include all the configured hard goals. 
+This allows for more flexibility in generating optimization proposals, but may lead to proposals that do not satisfy all hard goals.
+
+However, any hard goals included in the proposal-specific goals will still be treated as hard goals by Cruise Control, even with `skipHardGoalCheck: true`.
+
+[id='con-optimization-proposals-{context}']
+== Optimization proposals
+
+Optimization proposals are summaries of proposed changes based on the defined optimization goals, assessed in a specific order of priority. 
+You can approve or reject proposals and rerun them with adjusted goals if needed.
+
+With Cruise Control deployed for use in Strimzi, the process to generate and approve an optimization proposal is as follows:
+
+. Create a `KafkaRebalance` resource specifying optimization goals and any specific configurations. 
+This resource triggers Cruise Control to initiate the optimization process.
+. Strimzi Metrics Reporter runs in Kafka brokers, collecting raw metrics and publishing them to a dedicated Kafka topic (`strimzi.cruisecontrol.metrics`).
+Metrics for partition topics and modeling the impact of rebalances are collected in other xref:proc-cruise-control-auto-created-topics-{context}[topics automatically created when Cruise Control is deployed]. 
+. Load Monitor collects the metrics from Kafka brokers, including CPU, disk, and network utilization data, and sends load data to the Analyzer and Anomaly Detector.
```

Comment:
The load monitor is the component collecting, processing and storing the metrics mentioned in the previous sentence. It creates a cluster model of the load of the cluster which is used by the Analyzer and Anomaly Detector